### PR TITLE
Migrate from custom kotlinx-serializers for Java types to ks3

### PR DESCRIPTION
**Pull request by sschuberth**
Please have a look at the individual commit messages for the details.

### CODE REVIEW

**Comment by nnobelis** [comment]
Could you please give a bit of context on this?

Is there currently 3  JSON/Yaml serializers in ORT ?

- kotlinx
- ks3
- jackson

Is ks3 the new serializer we should always use now ?

File: clients/clearly-defined/src/main/kotlin/Curation.kt [hunk 1]
```diff
@@ -17,15 +17,12 @@
  * License-Filename: LICENSE
  */
 
-@file:UseSerializers(FileSerializer::class, URISerializer::class)
-
 package org.ossreviewtoolkit.clients.clearlydefined
 
-import java.io.File
-import java.net.URI
+import io.ks3.java.`typealias`.FileAsString
```
Comment (mnonnenmacher) [inline clients/clearly-defined/src/main/kotlin/Curation.kt:22]: That's a very inconvenient choice for a package name.

File: clients/clearly-defined/build.gradle.kts
Comment (mnonnenmacher) [inline clients/clearly-defined/build.gradle.kts:1]: Please add a link to the ks3 project / documentation to the commit message.

**Review by mnonnenmacher — CHANGES_REQUESTED** [review]

File: clients/clearly-defined/src/main/kotlin/Curation.kt (hunk 1 above)
Comment (sschuberth) [reply clients/clearly-defined/src/main/kotlin/Curation.kt:22]: I've [reported](https://github.com/Kantis/ks3/issues/113) it.

**Pull request merged**